.code-samples.meilisearch.yaml

```diff
@@ -251,12 +251,25 @@ search_parameter_guide_crop_1: |-
   SearchRequest searchRequest =
     new SearchRequest("shifu")
       .setAttributesToCrop(new String[] {"overview"})
-      .setCropLength(10);
+      .setCropLength(5);
+  client.index("movies").search(searchRequest);
+search_parameter_guide_crop_marker_1: |-
+  SearchRequest searchRequest =
+    new SearchRequest("shifu")
+      .setAttributesToCrop(new String[] {"overview"})
+      .setCropMarker("[…]");
   client.index("movies").search(searchRequest);
 search_parameter_guide_highlight_1: |-
   SearchRequest searchRequest =
     new SearchRequest("and").setAttributesToHighlight(new String[] {"overview"});
```

Comment:
This should be `"winter feast"` too
https://docs.meilisearch.com/reference/api/search.html#attributes-to-highlight